json-mock.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: json-mock
+  version: 1.3.8
+  epoch: 0
+  description: json mock server
+  dependencies:
+    runtime:
+      - bash # required by image entrypoint
+      - coreutils # env is required
+      - tini
+      - tini-compat
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/cilium/json-mock
+      tag: v${{package.version}}
+      expected-commit: c597f9fd3642da6a6b8143e77f06c76458434684
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}
+      cp run.sh default.json middleware.js ${{targets.contextdir}}
+
+test:
+  pipeline:
+    - name: stat on files
+      runs: |
+        stat /run.sh
```

Comment:
possible to run this script in someway to test it?

Reply:
thank you Mri for reviews, I've extended the tests via https://github.com/wolfi-dev/os/pull/41162/commits/83174e6fbe113f8154b31e9af717bed54781449e